src/components/common/Header.tsx

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import { PropsWithChildren } from "react";
+
+interface HeaderProps extends PropsWithChildren {
+  text: string;
+}
+
+export const Header = ({ text, children }: HeaderProps) => {
+  return (
+    <HeaderWrapper>
+      {children}
+      <Text>{text}</Text>
+    </HeaderWrapper>
+  );
+};
```

Comment:
PropsWithChildren 는 아래와 같이 제네릭으로 사용하시면 됩니다!

```suggestion
interface HeaderProps {
  text: string;
}

export const Header = ({ text, children }: PropsWithChildren<HeaderProps>) => {
  return (
    <HeaderWrapper>
      {children}
      <Text>{text}</Text>
    </HeaderWrapper>
  );
};
```